src/viam/sdk/robot/client.cpp

```diff
@@ -311,7 +311,9 @@ RobotClient::RobotClient(std::shared_ptr<ViamChannel> channel)
     : channel_(channel->channel()),
       viam_channel_(std::move(channel)),
       should_close_channel_(false),
-      impl_(std::make_unique<impl>(RobotService::NewStub(channel_))) {}
+      impl_(std::make_unique<impl>(RobotService::NewStub(channel_))) {
+    Registry::initialize();
+}
```

Comment:
(flyby) we need to ensure that the `Registry` is properly initialized even when functioning as a pure client. This isn't a great long-term fix but @acmorrow has some future work on library initialization that will help clear this up.